src/main/java/uk/gov/hmcts/dts/fact/controllers/admin/AdminRole.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.hmcts.dts.fact.controllers.admin;
+
+final class AdminRole {
```

Comment:
Don't think this class is ever used. If it it not used anywhere you can remove the class

Reply:
I think I move this class to be under "/services" at some stage because it is needed there too.

Reply:
Removed file and was not used anywhere